### PR TITLE
Deployment link added to PR template : Fixes #851

### LINKED ISSUE #851: The pull request template misses deployment link
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
The pull request template does mention of the deployment link generated by travis by surge. 

**Expected behavior:**
There should be a link template too. 

**Screenshots of the issue:**

Not applicable

Working on it

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Deployment link is now added to the pull request template itself.

Fixes #851 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added generated SURGE link
#### Changes proposed in this pull request:
- now a test link is added to the template itself.

- see [https://raw.githubusercontent.com/YashKumarVerma/susper.com/master/.github/PULL_REQUEST_TEMPLATE.md](https://raw.githubusercontent.com/YashKumarVerma/susper.com/master/.github/PULL_REQUEST_TEMPLATE.md)

### CODE REVIEW

**Comment by bhaveshAn** [comment]
From next time please, first make a new branch and then fix the issue, unlike this one.
`YashKumarVerma:master` to `fossasia:master`

**Comment by YashKumarVerma** [comment]
sure @bhaveshAn

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Fixes #
 -
 
 <!-- Demo Link: Add here the link where you changes can be seen. -->
+<!-- Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh -->
+[View Proposed Changes live](https://pr-PR_NUMBER-fossasia-susper.surge.sh)
```
Comment (vaibhavsingh97) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think this should be commented

Comment (vaibhavsingh97) [reply .github/PULL_REQUEST_TEMPLATE.md]: And add a checklist which says `Added genrated surge link`

**Comment by YashKumarVerma** [comment]
Done!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,13 +6,18 @@ Fixes #
 - [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
 - [ ] My branch is up-to-date with the Upstream `master` branch.
 - [ ] I have added necessary documentation (if appropriate)
+- [ ] Added Surge preview link
```
Comment (vaibhavsingh97) [inline .github/PULL_REQUEST_TEMPLATE.md]: @YashKumarVerma Add the commented block here `Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh`

**Comment by vaibhavsingh97** [comment]
Remove `[View Proposed Changes live](https://pr-PR_NUMBER-fossasia-susper.surge.sh)`

**Comment by YashKumarVerma** [comment]
@ParthS007 @vaibhavsingh97 
Apologies that i had been out of sync. Please see the changes

**Comment by YashKumarVerma** [comment]
@vaibhavsingh97

**Pull request merged**